meteor.js

```diff
@@ -49,21 +62,41 @@ var deleteFolderRecursive = function(path) {
     }
 };
 
+var deleteBuildFiles = function(buildPath) {
+    var files = [];
+    if (fs.existsSync(buildPath)) {
+        files = fs.readdirSync(buildPath);
+        files.forEach(function (file, index) {
+            var curPath = buildPath + "/" + file;
+            if (helpers.endsWith(file, ['.css', '.html', '.map', '.js'])) {
```

Comment:
Yes i think thats the best. IMO you should never put build files in a parent folder anyway.